### DISCUSSION TITLE
FroshPlatformThumbnailProcessor Lazy Load and custom template integration

### DISCUSSION THREAD

**Discussion by nikkoch** [FriendsOfShopware/FroshPlatformThumbnailProcessor#112] [Q&A] [answered]
Does FroshPlatformThumbnailProcessor support Lazy Load, if so can you recommend a solution please?

Can you also advice on integration in our custom theme template? 

Shopware 6

**Comment by tinect** [comment]
this plugin does no more template changes. Lazy load has been migrated to a dedicated plugin: [FroshLazySizes](https://github.com/FriendsOfShopware/FroshLazySizes)

**Comment by nikkoch** [comment]
thank you tinect!
What means 'thid plugin does no more template changes'?
Should we use standard Shopware 6 twig templating or some shortcodes or attributes for Frosh to be able to work with images?

**Reply by tinect** [reply, marked as answer]
with this plugin I mean: FroshPlatformThumbnailProcessor

use the plugin FroshLazySizes to add LazySizes and LazyLoad or implement it on your own.

but in fact, this topic has nothing to do with FroshPlatformThumbnailProcessor which does"just" manipulate URLs of esch image for APIs, so for storefront automatically.
